packages/embark/src/lib/core/engine.js

```diff
@@ -231,228 +220,6 @@ class Engine {
-    return;
-    // let self = this;
-    //
-    // this.registerModulePackage('embark-code-generator', {plugins: self.plugins, env: self.env});
-    //
-    // const generateCode = function (modifiedAssets) {
-    // // self.events.request("module:storage:onReady", () => {
-    // self.events.request("code-generator:embarkjs:build", () => {
-    // self.events.emit('code-generator-ready', modifiedAssets);
-    // });
-    // // });
-    // };
-    // const cargo = async.cargo((tasks, callback) => {
-    // const modifiedAssets = tasks.map(task => task.modifiedAsset).filter(asset => asset); // filter null elements
-    // generateCode(modifiedAssets);
-    // self.events.once('outputDone', callback);
-    // });
-    // const addToCargo = function (modifiedAsset) {
-    // cargo.push({modifiedAsset});
-    // };
-    //
-    // this.events.on('contractsDeployed', addToCargo);
-    // this.events.on('blockchainDisabled', addToCargo);
-    // this.events.on('asset-changed', addToCargo);
-  }
-
-  setupCompilerAndContractsManagerService(options) {
-    this.registerModulePackage('embark-compiler', {plugins: this.plugins, isCoverage: options.isCoverage});
-    this.registerModulePackage('embark-solidity', {ipc: this.ipc, useDashboard: this.useDashboard});
-    this.registerModulePackage('embark-vyper');
-    this.registerModulePackage('embark-contracts-manager', {plugins: this.plugins, compileOnceOnly: options.compileOnceOnly});
-  }
-
-  deploymentService(options) {
-    let self = this;
-
-    this.setupCompilerAndContractsManagerService(options);
-    this.registerModulePackage('embark-solidity', {ipc: self.ipc, useDashboard: this.useDashboard});
-    this.registerModulePackage('embark-vyper');
-    this.registerModulePackage('embark-profiler');
-    this.registerModulePackage('embark-deploy-tracker', {trackContracts: options.trackContracts});
-    this.registerModulePackage('embark-specialconfigs');
-    this.registerModulePackage('embark-console-listener', {ipc: self.ipc});
-    this.registerModulePackage('embark-deployment', {plugins: this.plugins, onlyCompile: options.onlyCompile});
-    this.registerModulePackage('embark-transaction-tracker');
-    this.registerModulePackage('embark-debugger');
-  }
-
-  fileWatchService() {
-    this.registerModulePackage('embark-watcher');
-    this.events.request('watcher:start');
-  }
-
-  cockpitService() {
-    this.registerModulePackage('embark-authenticator', {singleUseAuthToken: this.singleUseAuthToken});
-    this.registerModulePackage('embark-api', {plugins: this.plugins});
-  }
-
-  cockpitModules() {
-    this.registerModulePackage('embark-authenticator', {singleUseAuthToken: this.singleUseAuthToken});
-    this.registerModulePackage('embark-api', {plugins: this.plugins});
-    // Register logs for the cockpit console
-    this.events.request('process:logs:register', {processName: EMBARK_PROCESS_NAME, eventName: "log", silent: false, alwaysAlreadyLogged: true});
-  }
```

Comment:
Cockpit modules are still useful. They just got lost in the services functions, but you must keep them